web_app/assets/styles.css

```diff
@@ -86,180 +86,160 @@ code {
 /*------------------------------------------------------------------------------------*/
 /* Backgrounds/Bodies */
 
-.background {
-    padding-left: 5%;
-    padding-right: 5%;
-    background-color: gainsboro;
-}
-
 .content {
-    min-height: 100rem;
-    padding-top: 2em;
-    padding-bottom: 2em;
-    padding-left: 6em;
-    padding-right: 6em;
+    padding-top: 1rem;
+    padding-bottom: 2rem;
+    padding-left: 7rem;
+    padding-right: 7rem;
```

Comment:
Might be better to consolidate these into a single line, like:
```css
padding: 1rem 7rem 2rem;
```

Reply:
That would definitely make it look like I know what I was doing! Really, I always forget what the CSS ordering is for the single-line syntax.